Runtime/safe-ds/tests/data/_table/test__str__.py

```diff
@@ -0,0 +1,10 @@
+import pandas as pd
+from safe_ds.data import Row, Table
+
+
+def test_add_row_valid() -> None:
+    table1 = Table(pd.DataFrame(data={"col1": [1, 2, 1], "col2": [1, 2, 4]}))
+    row = Row(pd.Series(data=[5, 6]), table1.schema)
+
+    table1 = table1.add_row(row)
+    assert table1.schema == row.schema
```

Comment:
Wieso ist dieser Test in der `__str__` test Datei? Ihr greift hier doch nicht auf die `__str__` zu?